v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples.py

```diff
@@ -83,63 +73,82 @@ def read_annotation_dependencies(self):
             annotation_dependencies['gencode_mapping'] = hl.literal(
                 load_gencode(GENCODE_RELEASE, ''),
             )
-
         return annotation_dependencies
 
     def requires(self) -> list[luigi.Task]:
+        upstream_table_tasks = []
         if self.dataset_type.has_sample_lookup_table:
             # NB: the sample lookup table task has remapped and subsetted callset tasks as dependencies.
-            upstream_table_tasks = [
-                UpdateSampleLookupTableTask(
+            upstream_table_tasks.extend(
+                [
+                    UpdateSampleLookupTableTask(
+                        self.reference_genome,
+                        self.dataset_type,
+                        self.sample_type,
+                        callset_path,
+                        project_guid,
+                        project_remap_path,
+                        project_pedigree_path,
+                        self.ignore_missing_samples_when_subsetting,
+                        self.ignore_missing_samples_when_remapping,
+                        self.validate,
+                    )
+                    for (
+                        callset_path,
+                        project_guid,
+                        project_remap_path,
+                        project_pedigree_path,
+                    ) in callset_project_pairs(
+                        self.callset_paths,
+                        self.project_guids,
+                        self.project_remap_paths,
+                        self.project_pedigree_paths,
+                    )
+                ],
+            )
+        else:
+            upstream_table_tasks.extend(
+                [
+                    WriteRemappedAndSubsettedCallsetTask(
+                        self.reference_genome,
+                        self.dataset_type,
+                        self.sample_type,
+                        callset_path,
+                        project_guid,
+                        project_remap_path,
+                        project_pedigree_path,
+                        self.ignore_missing_samples_when_subsetting,
+                        self.ignore_missing_samples_when_remapping,
+                        self.validate,
+                    )
+                    for (
+                        callset_path,
+                        project_guid,
+                        project_remap_path,
+                        project_pedigree_path,
+                    ) in callset_project_pairs(
+                        self.callset_paths,
+                        self.project_guids,
+                        self.project_remap_paths,
+                        self.project_pedigree_paths,
+                    )
+                ],
+            )
+        upstream_table_tasks.extend(
+            [
+                UpdateVariantAnnotationsTableWithUpdatedReferenceDataset(
```

Comment:
this seems right to me, but on some local test runs I'm getting 
```
RuntimeError: Unfulfilled dependency at run time: UpdateVariantAnnotationsTableWithUpdatedReferenceDataset_SNV_INDEL_COMBINED_GRCh38_e9bde72efb
```
so I think the way I'm initializing these dependencies could be wrong